superset-frontend/src/datasource/DatasourceEditor.jsx

```diff
@@ -83,18 +83,23 @@ function ColumnCollectionTable({
               <Field
                 fieldKey="expression"
                 label={t('SQL Expression')}
-                control={<TextControl />}
+                control={
+                  <TextAreaControl
```

Comment:
That looked pretty big in the <tr>, would look bad if there was many of these stacked... We may want to inline it.

Reply:
update the sql expression field to half the size